mealie/services/scraper/cleaner.py

```diff
@@ -570,4 +583,10 @@ def clean_nutrition(nutrition: dict | None) -> dict[str, str]:
                 with contextlib.suppress(AttributeError, TypeError):
                     output_nutrition[key] = str(float(output_nutrition[key]) * 1000)
 
+    for key in ["calories"]:
+        if val := nutrition.get(key, None):
+            if isinstance(val, int):
```

Comment:
While we are at it lets get floats in there as well, can remember exactly where, but thats something i have seen somewhere

```suggestion
            if isinstance(val, int | float):
```